Documentation for Elastic Inference Service with Cloud Connect (#4694)
elastic/docs-content: explore-analyze/elastic-inference/connect-self-managed-cluster-to-eis.md

---
navigation_title: EIS for self-managed clusters
applies_to:
  stack: ga 9.3
  deployment:
    self: ga
    ece: ga
    eck: ga
---

# EIS for self-managed clusters

Elastic {{infer-cap}} Service (EIS) for self-managed clusters is available through [Cloud Connect](/deploy-manage/cloud-connect.md). Cloud Connect enables you to use {{ecloud}} services in your self-managed cluster without having to install and maintain their infrastructure yourself.

> You can use EIS for [semantic search](/solutions/search/semantic-search.md), [AI Assistant](/explore-analyze/ai-features/ai-chat-experiences/ai-assistant.md) and [AI Agent](/explore-analyze/ai-features/elastic-agent-builder.md), [Search Playground](/explore-analyze/query-filter/tools/playground.md), [Attack Discovery](/solutions/security/ai/attack-discovery.md), and [SIEM migrations](/solutions/security/get-started/automatic-migration.md).

Reviewer: ```suggestion
You can use EIS, for example, for [semantic search](/solutions/search/semantic-search.md), [AI Assistants](/explore-analyze/ai-features/ai-chat-experiences/ai-assistant.md) or [Attack Discovery](/solutions/security/ai/attack-discovery.md), among other use cases.
```
Same here

## Prerequisites

Before you can use EIS with your self-managed cluster, ensure you meet the following requirements:

* Your self-managed cluster is on an [Enterprise self-managed license]({{subscriptions}}) or an [active self-managed trial](https://cloud.elastic.co/registration)
* You have an {{ecloud}} account with either an [active Cloud Trial](https://cloud.elastic.co/registration) or [billing information configured](/deploy-manage/cloud-organization/billing/add-billing-details.md)

## Set up EIS with Cloud Connect

To set up EIS for your self-managed cluster with Cloud Connect:

:::::::{stepper}
::::::{step} Open Cloud Connect

> In your self-managed cluster, navigate to the **Cloud Connect** page using the [search bar](/explore-analyze/find-and-organize/find-apps-and-objects.md).

Reviewer: ```suggestion
In your self-managed Kibana instance, navigate to the **Cloud Connect** page using the [search bar](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```

:::{image} /explore-analyze/images/cloud-connect-eis.png
:screenshot:
:alt: Screenshot showing Cloud Connect page
:::

::::::

::::::{step} Get your Cloud Connect API key
Sign up or log in to {{ecloud}} and get the Cloud Connect API key:

- If you already have an {{ecloud}} account, click **Log in**.

> - If you don’t have an account yet, click **Sign up** and follow the prompts to create your account and start a free trial.

Reviewer: ```suggestion
- If you don’t have an account yet, click **Sign up** and follow the prompts to create your account and start a free Cloud Trial.
```
same here :)

---
navigation_title: EIS for self-managed clusters
applies_to:
  stack: ga 9.3
  deployment:
    self: ga
    ece: ga

>     eck: ga

Reviewer 2: ```suggestion
products:
  - id: cloud-enterprise
  - id: cloud-kubernetes
  - id: elastic-stack
description: Use Elastic Inference Service (EIS) with your self-managed, ECE, and ECK clusters thanks to Cloud Connect.
```
I wouldn't mix dimensions (stack version and deployment types) here. Instead, you already have clear pre-requisites, a clear title, and we should add "products" keys for the search, and a description
